pom.xml

```diff
@@ -35,7 +35,7 @@
         <url>https://issues.redhat.com/</url>
     </issueManagement>
 
-    <scm>
+    <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
```

Comment:
Do we know whether these attributes are inherited to child projects even if they have their own `<scm>` tag?

Reply:
Hi! Yes, I wrote a reproducer to verify this: https://github.com/lucifer4j/maven-urls . According to that if a child project overrides scm but does not set these attribute explicitly the parent's attributes are inherited.